docs/getting_started/creating_an_admin.rst

```diff
@@ -198,15 +204,17 @@ Step 4: Register SonataAdmin custom Routes
 SonataAdminBundle generates routes for the Admin classes on the fly. To load these
 routes, you have to make sure the routing loader of the SonataAdminBundle is executed:
 
-.. code-block:: yaml
+.. configuration-block::
+
+    .. code-block:: yaml
 
-    # app/config/routing.yml
+        # config/routing.yaml
```

Comment:
This should have been `routes.yaml` or `config/routes/sonata_admin.yaml`

Reply:
I guess I can't fix this here, I need to create a new PR on stable? /cc @jordisala1991

Reply:
Wow, I didn't saw it. IMO let's merge this as is, and fix it on stable when possible